my-tweet/src/context/post-context.js

```diff
@@ -13,7 +13,10 @@ const PostProvider=({children})=>{
     const [userUnfollow,setUserUnfollow]=useState({})
     const [loggedInUser,setLoggedInUser]=useState()
     const [editUserData,setEditUserData]=useState({firstName:"",lastName:"",username:"",bio:"",portfolioURL:""})
```

Comment:
you can use useReducer for manage multiple state so that your code looks more cleaner

Reply:
ok